phpunit.xml.dist

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/10.0/phpunit.xsd"
+         colors="true"
+         bootstrap="tests/bootstrap.php"
+         failOnRisky="true"
+         failOnWarning="true"
+         stderr="true">
+    <php>
+        <ini name="error_reporting" value="32767" />
+        <ini name="intl.default_locale" value="en" />
+        <ini name="intl.error_level" value="0" />
+        <ini name="memory_limit" value="-1" />
+        <ini name="apc.enable_cli" value="1" />
+
+        <env name="MEMCACHED_HOST" value="localhost" />
```

Comment:
I suggest configuring this environment variable on `.github/workflows/continuous-integration.yml`.

It will be hard to change its value on local environment, for example with docker.

Reply:
On local environment you can override the dist file with phpunit.xml. The main goal is to always run every test without skipping.

Reply:
`MEMCACHED_HOST` change depending on the infrastructure, but not depending on test things.

This variable should be configured on `docker-compose.yaml` or on `.env`.

To achieve that goal, do then, do not skip it on the test but make it fail.

Reply:
Oh i see. I missed your point. I guess we have to refactor our `docker-compose.yaml` later. The `docker-compose.yaml` is only for testing, but 1) we should make a test profile 2) alongside an "official" docker compose for dev/prod profile.

Reply:
I like the idea of an "official" docker compose for dev/prod profile.

The current `symfony/symfony` repository does not provide an "official" docker compose.
Maybe it will go on another repository like https://github.com/dunglas/symfony-docker.